shared/services/config/mev-boost-config.go

```diff
@@ -100,6 +100,20 @@ func NewMevBoostConfig(cfg *RocketPoolConfig) *MevBoostConfig {
 		relayMap[relay.ID] = relay
 	}
 
+	rpcPortModes := []config.ParameterOption{{
+		Name:        "Closed",
+		Description: "Do not allow connections to the RPC port",
+		Value:       config.RPC_Closed,
+	}, {
+		Name:        "Open to Localhost",
+		Description: "Allow connections from this host only",
+		Value:       config.RPC_OpenLocalhost,
+	}, {
+		Name:        "Open to External hosts",
+		Description: "Allow connections from external hosts. This is safe if you're running your node on your local network. If you're a VPS user, this would expose your node to the internet",
+		Value:       config.RPC_OpenExternal,
+	}}
+
```

Comment:
let's put this slice in a single place. It gets used quite a lot, might as well be DRY

Reply:
I wouldn't be opposed to breaking types/config/types.go out into types/config/ports.go or something, and sticking it there as
```golang
// RPCModes returns each supported RPCMode as a config.ParameterOption in a slice.
// If passed, warningOverride is used as the Description field of the config.ParameterOption for the 'external' setting.
func PortModes(warningOverride string) []config.ParameterOption {
...
}
```

Just make sure to return a new slice each time because otherwise you have to deal with pass-by-reference crap for warningOverride.

Reply:
Done.